### PR TITLE
Add Streaming inference Python example

### PR DESCRIPTION
**Pull request by qiyuangong**
Add streaming object detection Python example.
Add streaming text classification Python example.

### CODE REVIEW

**Comment by qiyuangong** [comment]
Passed Jenkins PR-Validation http://10.239.47.211:18888/job/ZOO-PR-Validation/2076/

File: pyzoo/zoo/examples/streaming/textclassification/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+1. TERMINAL 1: Running Netcat
+```
+nc -lk [port]
+```
+
+2. TERMINAL 2: Start StreamingTextClassification
+```
+MASTER=...
+embeddingPath=... // glove path. Local file system/HDFS/Amazon S3 are supported
```
Comment (hkvision) [inline pyzoo/zoo/examples/streaming/textclassification/README.md]: is this used?

File: pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    parser = OptionParser()
+    parser.add_option("--host", dest="host",
+                      default="localhost")
+    parser.add_option("--port", dest="port",
+                      default="9999")
+    parser.add_option("--index_path", dest="index_path")
+    parser.add_option("--partition_num", dest="partition_num",
+                      default="4")
+    parser.add_option("--token_length", dest="token_length",
+                      default="200")
```
Comment (hkvision) [inline pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: is this used?

**Comment by hkvision** [comment]
LGTM

File: pyzoo/zoo/examples/streaming/textclassification/README.md (hunk 1 above)
Comment (qiyuangong) [reply pyzoo/zoo/examples/streaming/textclassification/README.md]: Thanks. Already removed. :)

File: pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py (hunk 2 above)
Comment (qiyuangong) [reply pyzoo/zoo/examples/streaming/textclassification/streaming_text_classification.py]: No. It is not used. Already removed. :)

**Pull request merged**